components/Calendar.jsx

```diff
@@ -6,48 +6,60 @@ import "react-big-calendar/lib/css/react-big-calendar.css";
 import CustomToolbar from "./CustomToolbar.jsx";
 import CustomEvent from "./CustomEvent.jsx";
 import Modal from "./Modal.jsx";
+import axios from "axios";
 
 const mLocalizer = momentLocalizer(moment);
 
 const CalendarEvents = () => {
+  axios
+    .get(
+      `https://www.googleapis.com/calendar/v3/calendars/${process.env.NEXT_PUBLIC_GOOGLE_CALENDAR_EMAIL}/events?key=${process.env.NEXT_PUBLIC_GOOGLE_CALENDAR_API_KEY}&singleEvents=true&orderBy=starttime`
+    )
+    .then((response) => {
+      const result = JSON.stringify({ events: response.data.items });
```

Comment:
u don't need the JSON stuff, its alr in the proper format

Reply:
You do need the color coding part to get the colors correct, put a screenshot of the calendar events here

Reply:
> u don't need the JSON stuff, its alr in the proper format

When I don't add the JSON.stringify() it won't properly display the events in the control log since I'm trying to convert an event(a object in the array) to a string.

Reply:
> You do need the color coding part to get the colors correct, put a screenshot of the calendar events here

![image](https://github.com/acm-ucr/membership-portal/assets/79764515/e74c377e-90dc-43ac-8db9-67f582325bc4)

Reply:
> > u don't need the JSON stuff, its alr in the proper format
> 
> When I don't add the JSON.stringify() it won't properly display the events in the control log since I'm trying to convert an event(a object in the array) to a string.

can u show the results of response, response.data

Reply:
Both response and response.data display this(WITH JSON.stringify() )
![Screenshot 2023-08-28 152120](https://github.com/acm-ucr/membership-portal/assets/79764515/1632b69f-589b-4e8a-b09f-4adc4f77e230)

Reply:
without json.stringify() and only const result = response.data displays : 
![Screenshot 2023-08-28 152629](https://github.com/acm-ucr/membership-portal/assets/79764515/052c2b57-c07d-47fd-ae53-d353d56b6b63)

Reply:
and response only displays this:
![Screenshot 2023-08-28 153539](https://github.com/acm-ucr/membership-portal/assets/79764515/760b4c70-60f3-4323-ad8a-793a45fbab2c)